src/calculation/MAR_TC_003_calculatorVerify.java

```diff
@@ -0,0 +1,14 @@
+package calculation;
+
+public class MAR_TC_003_calculatorVerify {
+	public static void main(String[] args) {
+		
+		public void calculatorVerify() {
+			
+			
+		System.out.println("calculator is verified");
+		
```

Comment:
hi komal please add one more printing statement....

Reply:
hello manisha i have added the printing statement as per your guidance.
review the code and merge to master branch.